nifi-stateless/nifi-stateless-bundle/nifi-stateless-engine/src/main/java/org/apache/nifi/stateless/flow/StatelessFlowConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.stateless.flow;
+
+public class StatelessFlowConfiguration {
```

Comment:
I don't think this class is necessary. It's simply a wrapper around a long. But a `StatelessFlowConfiguration` would imply a great deal more configuration. We should instead just provide the timeout to the Stateless Flow

Reply:
Additional config elements can be passed in the constructor as well. Or we could use a Builder. Typically I'd recommend a Builder pattern after about 4-5 constructor args. We could also potentially provide a Configuration type of class like this in the future, if necessary. If this were a public API I think a "config" type of property might make more sense - but we would either want to rename it, or truly encapsulate all of the configuration elements for a Stateless Flow in it. But in this case, it's not a public API, so we can change it any time.